style/scss/helpers/_utility-classes.scss

```
@@ -1314,3 +1314,11 @@
     display: none;
   }
 }
+
+//// ----------------------------
+// Margin
+// ----------------------------
+
+.mt-6, .my-6 {
+  margin-top: 2rem!important;
+}
```

```suggestion
.mt-6,
.my-6 {
  margin-top: 2rem !important;
}

```

Can you avoid using `!important`? Higher specificity is usually preferable.

---

I matched how the bootstrap ones were written, I will remove it.

---

I think it's fine in that case, we can add comments to say that though in case devs wonder why it's there.